cfgov/cal/views.py

```diff
@@ -150,4 +217,5 @@ def pdf_response(request, context):
         response['Content-Disposition'] = 'attachment; filename=cfpb-leadership.pdf'
         return response
     except (urllib2.HTTPError, urllib2.URLError, BadStatusLine):
-        pass
+        messages.error(request, 'Error Creating PDF', extra_tags='leadership-calendar')
+        return HttpResponseRedirect('/the-bureau/leadership-calendar/')
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/pull/2263/commits/8b1241190c12d6ca8aef3094ad43dcefe3459d7e.